test/driver.js

```diff
@@ -879,8 +876,7 @@ class Driver {
                       annotationCanvasMap,
                       page,
                       IMAGE_RESOURCES_PATH,
-                      renderForms,
-                      this._l10n
```

Comment:
With this usage removed, can't we also remove the private member at https://github.com/mozilla/pdf.js/blob/b44387f8317d3c5283217352f3296f5aa226a7e6/test/driver.js#L356 altogether now since it only seems to be assigned but not used anymore?

Reply:
Well, it's still being used but it's more "indirect" now.

Note that we still need to *initialize* the `GenericL10n`, otherwise various `annotationLayer` tests will fail. With Fluent registered DOM-element are automatically translated, provided that there's an active `L10n`-instance; please see
https://github.com/mozilla/pdf.js/blob/898cc2e399e47ae70e08624a72852954030370b0/test/driver.js#L244-L264

Reply:
Ah, OK. In that case would it be an idea to add a comment at the assignment summarizing this, to avoid similar confusion later on as well, or in case it gets flagged by a static analyzer as "assigned but not read"?